### PR TITLE
add ISO language model

### PR DESCRIPTION
**Pull request by peterhunn**
New PR in lieu of #46

Signed-off-by: Peter Hunn <code@peterhunn.com>

### CODE REVIEW

File: language.cto [hunk 1]
```diff
@@ -0,0 +1,483 @@
+namespace org.accordproject.language
+
+/**
+ * Model to define the governing language
+ */
+ concept GoverningLanguage
```
Comment (dselman) [inline language.cto]: This contract specific concept doesn't belong in a general namespace like `org.accordproject.language`. Please remove from this file and introduce into a new namespace which imports the language type.

File: language.cto [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+  o NepalBhasa
+  o Nias
+  o Niger-KordofanianLanguages
+  o Niuean
+  o NorwegianNynorsk
+  o BokmålNorwegian
+  o Nogai
+  o Norse
+  o Norwegian
+  o N'Ko
```
Comment (dselman) [inline language.cto]: I'm surprised that `-` and `'` are valid here. Did you edit this is the VSCode extension for that it is checked?

File: language.cto [hunk 3]
```diff
@@ -0,0 +1,483 @@
...
+/**
+ * Model to define the governing language
+ */
+ concept GoverningLanguage
+  o Language inLanguage optional // Language in which the contract is written
+  o Language translatedLanguage optional // Language in which the contract is or may be translated to
+  o Language governingLanguage // Language that will govern the contract in the event of translation/multiple versions 
+
+/**
+ * ISO Language Codes (639-1 and 693-2)
```
Comment (dselman) [inline language.cto]: What are these exactly? ISO Language names?
https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

**Comment by jeromesimeon** [comment]
A couple of questions/comments:
- Could we put the URL to the source for that list as comment in the model?
- Should this use language codes rather than language names?

File: language.cto (hunk 3 above)
Comment (jeromesimeon) [reply language.cto]: https://datahub.io/core/language-codes (from @mttrbrts )

**Comment by jeromesimeon** [comment]
@peterhunn any progress on this?

**Comment by jeromesimeon** [comment]
@peterhunn should I close this PR, or are you still working on it?

**Comment by jeromesimeon** [comment]
Pinging @peterhunn about this model?

**Comment by peterhunn** [comment]
> Pinging @peterhunn about this model?

Are the updates suitable to render it usable?

**Comment by jeromesimeon** [comment]
> > Pinging @peterhunn about this model?
> 
> Are the updates suitable to render it usable?

I think at the minimum, @dselman comments have to be addressed and the DCO needs to be signed before we can merge.

**Comment by jeromesimeon** [comment]
> > Pinging @peterhunn about this model?
> 
> Are the updates suitable to render it usable?

Maybe the comments have been addressed? I'll ask Dan for another review to make sure. The only other change might be to sign the commit for DCO signoff.

**Comment by dselman** [comment]
Closing, as stale.

**Pull request closed without merging**